modules/nw-ovn-kubernetes-list-resources.adoc

```diff
@@ -25,75 +25,79 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
+NAME                          DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                 AGE
+daemonset.apps/ovnkube-node   6         6         6       6            6           beta.kubernetes.io/os=linux   114m
 
-NAME                              ENDPOINTS                                                        AGE
-endpoints/ovn-kubernetes-master   10.0.132.11:9102,10.0.151.18:9102,10.0.192.45:9102               57m
-endpoints/ovn-kubernetes-node     10.0.132.11:9105,10.0.143.72:9105,10.0.151.18:9105 + 7 more...   57m
-endpoints/ovnkube-db              10.0.132.11:9642,10.0.151.18:9642,10.0.192.45:9642 + 3 more...   57m
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/ovnkube-control-plane   3/3     3            3           114m
+
+NAME                                               DESIRED   CURRENT   READY   AGE
+replicaset.apps/ovnkube-control-plane-65c6f55656   3         3         3       114m
+
+NAME                                     ENDPOINTS                                               AGE
+endpoints/ovn-kubernetes-control-plane   10.0.0.3:9106,10.0.0.4:9106,10.0.0.5:9106               114m
+endpoints/ovn-kubernetes-node            10.0.0.3:9105,10.0.0.4:9105,10.0.0.5:9105 + 9 more...   114m
 
 NAME                                 DATA   AGE
-configmap/control-plane-status       1      55m
-configmap/kube-root-ca.crt           1      57m
-configmap/openshift-service-ca.crt   1      57m
-configmap/ovn-ca                     1      57m
-configmap/ovn-kubernetes-master      0      55m
-configmap/ovnkube-config             1      57m
-configmap/signer-ca                  1      57m
+configmap/control-plane-status       1      113m
+configmap/kube-root-ca.crt           1      114m
+configmap/openshift-service-ca.crt   1      114m
+configmap/ovn-ca                     1      114m
+configmap/ovnkube-config             1      114m
+configmap/signer-ca                  1      114m
+
 ----
 +
-There are three `ovnkube-masters` that run on the control plane nodes, and two daemon sets used to deploy the `ovnkube-master` and `ovnkube-node` pods.
-There is one `ovnkube-node` pod for each node in the cluster.
-In this example, there are 5, and since there is one `ovnkube-node` per node in the cluster, there are five nodes in the cluster.
-The `ovnkube-config` `ConfigMap` has the {product-title} OVN-Kubernetes configurations started by online-master and `ovnkube-node`.
-The `ovn-kubernetes-master` `ConfigMap` has the information of the current online master leader.
+There are three `ovnkube-cluster-manager` that run on the control plane nodes, and one daemon set used to deploy the `ovnkube-control-plane` and `ovnkube-node` pods.
+There is one ovnkube-node pod for each node in the cluster.
+The `ovnkube-config` `ConfigMap` has the {product-title} OVN-Kubernetes configurations.
++
+[NOTE]
+====
+For Azure, there will be eleven `ovnkube-node` pods in a cluster.
```

Comment:
ah this also makes no sense. What I meant was there were 10 containers and it was 11 in azure per ovnkube-node pod, now its become 8 containers as seen in your output, which means 9 containers on azure. Please remove this NOTE, I did not mean to ask you to make it part of docs :D I was just commenting.

We just mention there is one ovnkube-node pod per node, then how can there be 11 pods? :) Let's remove the `NOTE`

Reply:
Ok, I now understand what you were saying previously. Thank you.